app/Models/Application.php

```
@@ -1555,40 +1556,185 @@ protected function buildGitCheckoutCommand($target): string
         return $command;
     }
 
+    private function parseWatchPaths($value)
+    {
+        if ($value) {
+            $watch_paths = collect(explode("\n", $value))
```

PHPMD (2.15.0)

The variable $watch_paths is not named in camelCase. (undefined)

(CamelCaseVariableName)